src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/CollectDataForReadOrientationFilter.java

```diff
@@ -196,13 +196,11 @@ private boolean isPileupGood(final ReadPileup pileup, final ReferenceContext ref
 
         List<Integer> mappingQualities = Ints.asList(pileup.getMappingQuals());
 
-        final boolean isIndel = pileup.getNumberOfElements(pe -> pe.isDeletion() || pe.isAfterInsertion() || pe.isBeforeDeletionStart()) > 0;
+        boolean isIndel = pileup.getNumberOfElements(pe -> pe.isDeletion() || pe.isAfterInsertion() || pe.isBeforeDeletionStart()) > 0;
```

Comment:
`pileup.getNumberOFElements(. . .)` is relatively expensive, so you should `return true` at the beginning if `pileup.size() > 0`.  Also, instead of counting *all* elements, you only need to find one with `Utils.stream(pileup).anyMatch(pe -> pe.isDeletion() || pe.isAfterInsertion() || pe.isBeforeDeletionStart())`.